### PR TITLE
[function] add sqrt math function

### PR DESCRIPTION
**Pull request by Junnplus**
I hereby agree to the terms of the CLA available at: https://databend.rs/policies/cla/

## Summary

```
mysql> SELECT SQRT(16);
+----------+
| SQRT(16) |
+----------+
|        4 |
+----------+
1 row in set (0.00 sec)
Read 1 rows, 1 B in 0.000 sec., 3.9 thousand rows/sec., 3.9 KB/sec.

mysql> SELECT SQRT(0);
+---------+
| SQRT(0) |
+---------+
|       0 |
+---------+
1 row in set (0.00 sec)
Read 1 rows, 1 B in 0.000 sec., 2.35 thousand rows/sec., 2.35 KB/sec.

mysql> SELECT SQRT(-4);
+-------------+
| SQRT((- 4)) |
+-------------+
|        NULL |
+-------------+
1 row in set (0.00 sec)
Read 1 rows, 1 B in 0.000 sec., 2.82 thousand rows/sec., 2.82 KB/sec.
```

## Changelog

- New Feature

## Related Issues

https://github.com/datafuselabs/databend/issues/2563

## Test Plan

Unit Tests

Stateless Tests

### CODE REVIEW

File: common/functions/src/scalars/maths/sqrt.rs [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+use crate::scalars::function_factory::FunctionFeatures;
+use crate::scalars::Function;
+
+#[derive(Clone)]
+pub struct SqrtFunction {
+    _display_name: String,
+}
+
+impl SqrtFunction {
+    pub fn try_create(_display_name: &str) -> Result<Box<dyn Function>> {
```
Comment (BohuTANG) [inline common/functions/src/scalars/maths/sqrt.rs:33]: _display_name -> display_name

File: tests/suites/0_stateless/02_0014_function_maths.sql [hunk 2]
```diff
@@ -50,5 +50,8 @@ SELECT atan2(NULL, 2);
 SELECT atan2(NULL, NULL);
 SELECT atan2(NULL, number) from numbers(2);
 SELECT atan2(number, NULL) from numbers(2);
+SELECT sqrt(4);
+SELECT sqrt(0);
+SELECT sqrt(-4);
```
Comment (BohuTANG) [inline tests/suites/0_stateless/02_0014_function_maths.sql:55]: select sqrt('a');
result is: 0
http://sqlfiddle.com/#!9/9eecb/256797

**Review by zhyass — APPROVED** [review]
LGTM

**Pull request merged**